app/wizards/a_levels_wizard_store.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class ALevelsWizardStore < DfE::Wizard::Store
+  delegate :valid_step?, :current_step_name, :course, to: :wizard
+
+  def save
+    return false unless valid_step?
+
+    if current_step_name == :are_any_a_levels_required_for_this_course && current_step.answer == 'no'
+      course.update!(
+        a_level_requirements: false
+      )
```

Comment:
This is something I spotted in both Apply and Publish.

The next feature I will add on the wizard is the possibility to have a storage class for each step (if is defined). For now we have one central storage for the whole wizard. I will leave as it is until we support that feature in the next dfe-wizard gem version